app/models/planting.rb

```diff
@@ -137,4 +124,31 @@ def percentage_grown(current_date = Date.current)
 
     percent
   end
+
+  def start_to_finish_diff
+    (finished_at - planted_at).to_i
+  end
+
+  def self.mean_days_until_maturity(plantings)
+    ## Given a set of finished plantings, calculate the average/mean time from start to finish
+    differences = plantings.collect(&:start_to_finish_diff)
+
+    return if differences.compact.empty?
+
+    differences.compact.sum / differences.compact.size unless differences.compact.empty?
```

Comment:
i think also it should try to work out the days_until _maturity from the user's own plantings first -- and if there in't enough data, try people in nearby climates, and then finally resort to using the entire dataset.

People in Singapore will have different results to me in Whanganui-a-tara